#! /usr/bin/env python

PACKAGE='autorally_control'
import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
#       Name       Type      Level Description     Default Min   Max
gen.add("pGain", double_t,    0,    "Proportional gain", -2.0, -10, 10)
gen.add("gpsHeading",       bool_t,    0,    "True to use gps heading, false for fused", False)
gen.add("wpRadius",       double_t,    0,    "Distance from waypoint to move to next", 1.5,     0.5, 10)
gen.add("wpnum",       double_t,    0,    "Distance from waypoint to move to next", 1.5,     0.5, 10)

exit(gen.generate(PACKAGE, "gpsWaypoint", "gpsWaypoint_params"))
